main.go

```diff
@@ -4,29 +4,36 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"math/rand"
 	"os"
 	"strings"
 	"sync"
 	"time"
 
-	shell "github.com/whyrusleeping/pinbot/Godeps/_workspace/src/github.com/ipfs/go-ipfs-api"
-	hb "github.com/whyrusleeping/pinbot/Godeps/_workspace/src/github.com/whyrusleeping/hellabot"
+	shell "github.com/ipfs/go-ipfs-api"
+	hb "github.com/whyrusleeping/hellabot"
 )
 
 var prefix string
 var gateway string
 
 var (
-	cmdBotsnack = prefix + "botsnack"
```

Comment:
note that prefix here is empty, seemed like a left-over from the time a custom prefix flag was added.